module/VuFind/src/VuFind/ILS/Driver/XCNCIP2.php

```diff
@@ -295,6 +295,17 @@ class XCNCIP2 extends AbstractBase implements
      */
     protected $maxNumberOfPages;
 
+    /**
+     * Some ItemUseRestrictionType values could be useful as status. This property controls which values from
+     * ItemRestrictionType should replace the status value in response of getHolding method.
+     *
+     * @var array
+     */
+    protected $itemUseRestrictionType2Status = [
```

Comment:
I wonder if this variable name could be made more clear. I wondered at first what the "2" meant -- and if it's meant to represent "to" then that implies that this is a map, when it is not. Maybe `$itemUseRestrictionTypesForStatus`?

Also, would it make sense to make this configurable via XCNCIP2.ini for easier customization?